content/ja/docs/reference/access-authn-authz/authentication.md

```diff
@@ -341,65 +369,154 @@ current-context: webhook
+    # トークンに対する有効なオーディエンスとこのリストの共通部分をレスポンスステータスに
+    # 返却する必要があります。
+    # これにより、トークンが提示されたサーバーへの認証に有効であることが保障されます。
+    # オーディエンスが指定されていない場合、Kubernetes APIサーバーへの認証のために
+    # トークンを検証する必要があります。
+    "audiences": ["https://myserver.example.com", "https://myserver.internal.example.com"]
   }
 }
 ```
+{{% /tab %}}
+{{< /tabs >}}
 
-リモートサービスはログインの成功を示すために、リクエストの`status`フィールドを埋めることが期待されます。レスポンスボディの`spec`フィールドは無視され、省略することができます。Bearerトークンの検証に成功すると、以下のようにBearerトークンが返されます。
+リモートサービスはログインの成功を示すために、リクエストの`status`フィールドを埋めることが期待されます。レスポンスボディの`spec`フィールドは無視され、省略することができます。リモートサービスは、トークンが受信したものと同じ`TokenReview` APIバージョンを使用して、レスポンスを返却する必要があります。Bearerトークンの検証に成功すると、以下のようにBearerトークンが返されます。
 
-```json
+{{< tabs name="TokenReview_response_success" >}}
+{{% tab name="authentication.k8s.io/v1" %}}
+```yaml
+{
+  "apiVersion": "authentication.k8s.io/v1",
+  "kind": "TokenReview",
+  "status": {
+    "authenticated": true,
+    "user": {
+      # 必須
+      "username": "janedoe@example.com",
+      # オプション
+      "uid": "42",
+      # オプションのグループメンバーシップ
+      "groups": ["developers", "qa"],
+      # 認証機能が提供するオプションの追加情報です。
+      # これは、ログやAPIオブジェクトに記録されたり、アドミッションのWebhookで
+      # 利用可能になったりするため、機密データを記載するべきではないです。
+      "extra": {
+        "extrafield1": [
+          "extravalue1",
+          "extravalue2"
+        ]
+      }
+    },
+    # オーディエンス対応トークン認証機能が応答できるオプションリストに、
+    # 提供されたトークンが有効だった`spec.audiences`リストのオーディエンスを含みます。
+    # これが省略された場合、トークンはKubernetes APIサーバーへの認証に有効であるとみなされます。
+    "audiences": ["https://myserver.example.com"]
+  }
+}
+```
+{{% /tab %}}
+{{% tab name="authentication.k8s.io/v1beta1" %}}
+```yaml
 {
   "apiVersion": "authentication.k8s.io/v1beta1",
   "kind": "TokenReview",
   "status": {
     "authenticated": true,
     "user": {
+      # 必須
       "username": "janedoe@example.com",
+      # オプション
       "uid": "42",
-      "groups": [
-        "developers",
-        "qa"
-      ],
+      # オプショングループメンバーシップ
+      "groups": ["developers", "qa"],
+      # 認証機能が提供するオプションの追加情報です。
+      # これは、ログやAPIオブジェクトに記録されたり、アドミッションのWebhookで
+      # 利用可能になったりするため、機密データを記載するべきではないです。
       "extra": {
         "extrafield1": [
           "extravalue1",
           "extravalue2"
         ]
       }
-    }
+    },
+    # オーディエンス対応トークン認証機能が応答できるオプションリストに、
+    # 提供されたトークンが有効だった`spec.audiences`リストのオーディエンスを含みます。
+    # これが省略された場合、トークンはKubernetes APIサーバーへの認証に有効であるとみなされます。
+    "audiences": ["https://myserver.example.com"]
   }
 }
 ```
+{{% /tab %}}
+{{< /tabs >}}
 
 リクエストに失敗した場合は、以下のように返されます。
 
-```json
+{{< tabs name="TokenReview_response_error" >}}
+{{% tab name="authentication.k8s.io/v1" %}}
+```yaml
 {
   "apiVersion": "authentication.k8s.io/v1beta1",
   "kind": "TokenReview",
   "status": {
-    "authenticated": false
+    "authenticated": false,
+    # 認証に失敗した詳細をオプションで記載します。
+    # エラーが提供されていない場合、APIは一般的なUnauthorizedメッセージを返却します。
+    # authenticated=trueの場合、エラーフィールドは無視されます。
+    "error": "Credentials are expired"
   }
```

Comment:
You need to add sections about [here](https://gist.github.com/inductor/18a9278caa84436a6ee82cc80f12c38f#file-en-diff-L296-L309)?

Reply:
Fixed !